tensorflow_lite_support/tools/Build_TensorFlow_Lite_Support_Libraries_with_Bazel_and_Colab.ipynb

```diff
@@ -0,0 +1,243 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.        --host_crosstool_top=@bazel_tools//tools/cpp:toolchain \
</pre><p>This is default in our bazelrc as  well.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/tflite-support/pull/620/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
Removed